job-server/src/main/scala/spark/jobserver/JobCache.scala

```diff
@@ -28,7 +31,7 @@ class JobCacheImpl(maxEntries: Int,
 
   private val cache = new LRUCache[(String, DateTime, String, BinaryType), BinaryJobInfo](maxEntries)
   private val logger = LoggerFactory.getLogger(getClass)
-  implicit val daoAskTimeout: Timeout = Timeout(3 seconds)
+  implicit val daoAskTimeout: Timeout = Timeout(60 seconds)
```

Comment:
As above, this `val jarBinaryReq = (dao ? JobDAOActor.GetBinaryContent(appName, BinaryType.Jar, uploadTime))
          .mapTo[JobDAOActor.BinaryContent]` in JobCache often cause timeout error, 3 seconds is not enough.